### PR TITLE
Feature/profile 161 mj

### PR DESCRIPTION
**Pull request by MellowJay**
User logged in state is saved in the Login class boolean variable isUserLoggedIn.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,9 +36,26 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-            <version>1.4.196</version>
+            <version>1.4.199</version>
         </dependency>
-<!--        <dependency>-->
+        <dependency>
```
Comment (kryzma) [inline pom.xml:41]: Do we need Junit? 
If not is it bad if we have unused libraries?

File: src/main/java/com/psk/pyragas/ToolRent/test/TestPersistence.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import com.psk.pyragas.ToolRent.dao.*;
+import com.psk.pyragas.ToolRent.entities.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.transaction.Transactional;
+
```
Comment (kryzma) [inline src/main/java/com/psk/pyragas/ToolRent/test/TestPersistence.java:11]: Is this file to be deleted in future? (from name "Test")
If yes should files like these be in develop branch?

**Comment by kryzma** [comment]
Some commit names are uninformative ("it works"), maybe they should be more meaningful? Because if person wants to go through commit history to find mistake it might be difficult.
We can discus commit mater in meeting.


Overall code is compact, readable and works great👍

**Review by kryzma — CHANGES_REQUESTED** [review]
Some commit names are uninformative ("it works"), maybe they should be more meaningful? Because if person wants to go through commit history to find mistake it might be difficult.
We can discus commit mater in meeting.

Overall code is compact, readable and works great👍

File: pom.xml (hunk 1 above)
Comment (MellowJay) [reply pom.xml:40]: I guess no. If we need, we'll put it back.

File: src/main/java/com/psk/pyragas/ToolRent/test/TestPersistence.java (hunk 2 above)
Comment (MellowJay) [reply src/main/java/com/psk/pyragas/ToolRent/test/TestPersistence.java]: Yeah, it doesn't work as a test class. I'll delete it.

**Review by kryzma — APPROVED** [review]

**Pull request merged**